### DISCUSSION TITLE
DDOS defenses

### DISCUSSION THREAD

**Discussion by ibsusu** [uNetworking/uWebSockets#1146] [Q&A] [answered]
uWebSockets is secure from an encryption standpoint but I was wondering about DDOS reflection attacks and whatnot.  I'm worried about people spoofing their ips when I use this as an http server.  Would this be a valid concern?  Am I being stupid?

**Comment by ghost** [comment, marked as answer]
I think the most common way of dealing with DDOS is having a scalable architecture that can spin up as many instances needed to handle the load. Maybe cloudflare will do the trick by simply refusing the traffic?

**Comment by ibsusu** [comment]
It seems that what I was wondering about, TCP SYN-ACK flooding and reflection should be solved on the linux box itself rather than the server.  Everything else is volumetric which is just getting more boxes.

I can't use cloudflare though for my project though, they will charge for the amount of bandwidth I need to use.

Thanks for your input and sorry for the unnecessary troubles.
